tidb-lightning/monitor-tidb-lightning.md

```diff
@@ -44,14 +24,21 @@ pprof-port = 8289
 ...
 ```
 
-You need to configure Prometheus to make it discover the `tidb-lightning` server. For instance, you can directly add the server address to the `scrape_configs` section:
+and in `tikv-importer.toml`:
+
+```toml
+# Listening address of the status server.
+status-server-address = '0.0.0.0:8286'
+```
+
+You need to configure Prometheus to make it discover the servers. For instance, you can directly add the server address to the `scrape_configs` section:
 
 ```yaml
 ...
 scrape_configs:
   - job_name: 'tidb-lightning'
     static_configs:
-      - targets: ['192.168.20.10:8289']
+      - targets: ['192.168.20.10:8289', '192.168.20.9:8286']
```

Comment:
please modify scrape to:
`  - job_name: "tikv-importer"
    static_configs:
    - targets: ['172.16.5.169:8286']
  - job_name: 'lightning'
    static_configs:
    - targets: ['172.16.5.169:8289']`